Sources/RichTextKit/Fonts/StandardFontSizeProvider.swift

```diff
@@ -42,11 +42,10 @@ public extension StandardFontSizeProvider {
      */
     static var standardRichTextFontSize: CGFloat {
         get { StandardFontSizeProviderStorage.standardRichTextFontSize }
-        set { StandardFontSizeProviderStorage.standardRichTextFontSize = newValue }
```

Comment:
This is actually an API breaking change, since the setter is public. It's used to allow people to change the standard font size...although this should probably be provided as a configuration when creating the text editor.